### ISSUE TITLE
Keyboard Shortcuts?

### ISSUE THREAD

**Issue by cmivxx** [JasonLG1979/gnome-shell-extensions-mediaplayer#437]
So, this really isn't an issue, but I don't know how to contact you outside of this..

**LOVE the extension**...  This is now part of my Gnome Toolkit.  I am however a "largely" keyboard user, but my ThinkPad x250 doesn't have any dedicated media buttons (or Fn media buttons) so my only way to navigate songs is via the extension interface or Spotify itself.  I have most of the keyboard shortcuts built into gnome tweaked to my liking, but I would really love the ability to specify custom media keys. Is this currently possible and I just haven't found it yet?  Or is it something that can be built in?

Like I said, great stuff...   Keep up the good work!

**Comment by JasonLG1979** [comment]
Mediakeys are out of scope of the extension. Mediakeys are between GNOME and the media player. GNOME as of 3.20(I think?) already binds mediakeys to MPRIS automatically anyway. Meaning even if your player of choice doesn't have real mediakeys support the keys you specify as mediakeys on your keyboard should already work. Adding mediakeys to the extension would be redundant.
